From: Stefano Rivera <stefanor@debian.org>
Date: Sun, 18 Feb 2018 18:21:41 -0800
Subject: Remove dependency on python-graphviz

We reduce this to a Recommends, it's only used by Jupyter Notebooks

Forwarded: not-needed
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index f28ca18..0325d00 100755
--- a/setup.py
+++ b/setup.py
@@ -80,7 +80,6 @@ setup(
     keywords='object graph visualization graphviz garbage collection',
     py_modules=['objgraph'],
     install_requires=[
-        'graphviz',  # just for ipython support currently
     ],
     tests_require=['mock;python_version=="2.7"'],
     test_suite='tests.test_suite',
